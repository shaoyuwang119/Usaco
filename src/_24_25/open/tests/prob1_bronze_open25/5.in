96 94
D
WD
LDD
LDDD
LLLWD
LDLDLD
DDWDLDD
WWDDWLWD
LWLLDLDLD
WDWLDLLWWD
WWLLWWLWWLD
WLWLWDWDDLWD
DLLLDWWWDDDDD
LLWDWLLLWDDLLD
LWDWWDWLLWLLLDD
WDDDLWLWLWDDWDDD
WLWLLDLLWLDWDDWLD
DDWWDLWWWLLLWWDLDD
LDDWLLWDLWLLLDLDDLD
LDLLLDLDDDLLDDWLLWDD
DLDWLWDDWLLWWLDLLWLLD
WDLWDWLWLWDDDWLLWDLWLD
DDWLWDLWWWDLLWWLWWLDDLD
DWLWDLLWLDDLWLWDWDDDWWDD
LLWWDWLLDLWWLWDLDDWWWWWWD
LDLDDLLDWLLLLLDLDDWWLWDWWD
LLWWWWDDDDLWDLDWWWWDLLDWDLD
DWWWLWDDWWWWDDWDWWLLLWLDLLLD
LLWWLDWLLWDLWWLWWDDWWLDDLWDLD
WDLLDWDLLDWWDDWWWLLLLWWLLLLWDD
LWLDDDDLDLDLWDWLDLDLDDDLWWDLDDD
LLLWWWLLDWDWDDWDWDDWDDWLWWWLWLLD
DWWDLWLWDDLWWLDDDDWLWDWDWLDWDDDWD
WLWLWWLWLDLDLLDDDWLDLLWDDWWWLWLLLD
DLDWLWWLDWWDLWWLLLWWWDLLDWWDLWLWDWD
LLWWDLWLLWLDWWDWWWLLWDDWWDWWWLLDDDDD
WWLLLLLLDLLDLWWDLDDWLDLDWWDWWLDLLDWLD
LLDDLWWLLDLDDWWLLDWLLWDDLWWLDLDWWDDWDD
DDWDWDDLDWDDWLLWWLLLWWDDWWWDDWWDLLLLDDD
DLWWWWWWLWWLLDWDLLDWDLWLLDWDLDWWDLLDWWDD
LLWLDWWLWWWWDLLWLDWLLWDWDDLLWWDLWWWWLLWWD
LDLWWWLLDDLDDWLWWWWLLDLWDWWWLLDDLLLDWWDWLD
DWDWWDDDDLLDLDWLLLWWLDDWLWLWWDWWDDWDWDWWLLD
LDDWDWDWLDLDWWDDDDWLLLWWDWDLWLLDLWLWWWWDDDDD
DLLLLLWDLDWDLWLLLWWLDDLLDWWWLLDDDDDLLWDLDLDLD
LDDDLDWWDWWWLDWDDWWLLDDWDLLWWLWDDDWLWLDDLWLDDD
WWDWLDLWDLWLWLDLDWLLDDLLLWDLDLWDWLDWDDDWLWLWLLD
WWWWDDLLWDLLWLWLWWWLLLDWWWDWWWWWDWLLWDLLWLDWDWLD
WDDLWDWDDLLLWDWDLDLWDLWWWWLLLWLWDDLDDDLDDWLWLLDWD
WDWDLWLWDDLWLWWDLWDDDLDWDDWWDDDWDLDLDLLWDLLWDLDLWD
DWWDDDWWDLDDLWWDWLLWWLDLWWWWDWWLWLWWLDDWLWDLDDLWDLD
DWWDDDWDWLDWWDWWLDDWDWLDLDLLDWDDDLDWDDLWWLLWWWDLWDLD
WDLDDDDWWDLWLLLWLDDLDDLWDLLLWLLDWLLWDLDWLDDDLLWLDLDDD
DLWWLWDWLLDWWDWLLDDWDLDLLLLDLLLDWWWWDLLWLWLWLDLDWDWLLD
LDWWDDLDLDWDLLDDDWDLWDLDLWWLDWDDWLLDLDWLLWDDWDDDLLLWDLD
WWLDWLDWLDWLWLDDDWLWDDLWLDDWLDWWLDDLWDWWLWWDDDLDDWLWWWLD
WDWLLLWWDWWWDWLWLWLDDLWDLWLLWDLDDLWWLLLWDWWDWWDDLLLDLLWDD
LWDWLWWLDDWWWLLDLWLLDDLLLWLWLDDLLLLLDDDLLDLLLWLDDLWDWDLWDD
DWDLWWWLWWDDWLLLWWWDWDWDWLWDWWWLLLLLWWLWWLDLDWWWLLDLDDWLLWD
WLWDLWDDLWLWWWLDLWLLDLDDLDWDWDWDWWWWWDDWLDDLLLDLLLWWWLWLLLDD
DLWDDWDDWDLDDDLWWDWDDLWWWWDWLDWWLWDDWDWDLWDDDDWWWLWLLWDWWLLWD
LWWWDWWWWWWWLWLWDWLDWWDDLLWLWDLWWWLLLLWDWDLDLDDWDWWLLDDDLWDWLD
LDWWDLLWWDDDLWDLLWLLDLDWDDLWWLDDWLWWWWDWDDWWLWWWLWWDWLWLLLLLWWD
WLDWDDLDDDDDLWWWDLWDDDWWWWDWDDDWDWLLDDWWWWDWWDWDLDDLWWDLDDWLDLDD
DWLDDLWWWDLWLDLLLWDLDWLWLDLDLWWDDDWLWDDWDDWLWLWDLDLWLLDWDLDWLLLLD
LWWLWDWWWLDLLLDLWWWWDWLWWWLDWLWLDLLWWDWDLLDDWDWLWWLWWWDWLDWDLDDLWD
DWWDDWDWWLDDWLWWDWLDDDLLWDDWDDLLLDWDDWDLDWDLDWWWWWWDLWDLDDWDDDDWDWD
DDWDWDLDWWLDLLWLWDWWLWWLLLWLDWWLDLWDLWWLWDLLWDLDDDWWLWLWWLLWDWLWDDWD
LDLWDWLWDWDDLLWLDLLDLLLDDLLWWDDLWWDDLLDLLLDDDLDDWDDDWDLDWLWLDLDWWDWDD
DDDWDDLWDLLWDWWWWWWWDLDDLDWWWWLDLWWWLDLLLLWLLDDWWDDLDWWLDDDWLDDLWLDDWD
DDWLDLLLLLDDDLDLDWLLLWLWLLLWLWWDDLDWDDLLDWDLLWDWDLDWWDWLLDLWWDDLLLDDWLD
DWDWDDDWWDLDLLWWWLWDDDWWWLWWWDWDDDLDDWLDLWLWDLLDLLDDLDLDLWLDDDWDWWDWLDDD
DDWWWDWWDWLDLLLWDLLLWLLDLWWLLWWLWWLDDWLLWWWLLWWWWLWDWDLWDDLDDWWWDDLWDDDLD
WDLDDWWDLDWLLDDWWDWDDLWDDWWDLDLWWWDLWDWDWLWLLWWDWWDLWDDLDDDWLWLDDDLWWLDWWD
WLWWDWWLLLWDDLLLDLDLLDDDDDLWWDDLDWWLLWDLLDDLLDWWWWDWDDDLDLWDWDDLDLWDDLDLWLD
LDWLWWWLDLDDLLLLWLDLDLLDLWLLWDLLLWWDLDLDWWLDWDWWWLLWWLDWDLWLDDDLDLWLLLDLLDLD
LDLDLDDDLLLDLDLWDWWWLDWDLLWDDDWWLLWLWDDDDDWDWLWWDWLLWWDLDDDDDDWDDDLWLDLDLWDLD
DWWLLLDWLWDDLDLLWDLDLLLWLDWWWWWLWWDDDDLWDDWLWLDDDLLLWDLDLLWDLWWWDLDDDLLWLDWDLD
WDLWWDLWDLDDWLWDLDLWLLWWLLWDLWLWLDLLWWDLWWLLLLLWWLWLWLDLDLWWWWLDLWWLDLWLDDLDWLD
LLLLLWDDDLDWDLLLLLWDLLDWWWLLWWLDLDDWWDWDWLDLWWDDWLDWWLWWLLDLDWDLLWLLWDDWLWWLLWLD
DWDWDWDLLWDLWWDWWWDDDLWWDWLDLWDLWLLLLLDWWLLWDWLLLLWLWDWDLLDDWDDWLDWWWDWWLWLDLLDDD
WWWWDDWLWLWWWDDLWDDWDDWWDDDWWDLWLDWDWWLDLWWDLLDWLLWLWDLDDLWWWWDLLWWLWDWLDWLWWDLDDD
LDWLWLDWLWWLLWWDDWWLDDDWLWLDWWDDWWLDDLWWWWLDWWDWLWWWDLDDLDDLDLLWLWDDLDWDDLLWDDWWWWD
DWLWLDLLDLDDDWLLLWDDWWWDLLWLWWDDWDDDDLLWDDWWLWDWWWWWDDWWLLDWDLLWLWLWDDWDLWDWLLLLDLWD
LDWDWDDLWLDLDWLLDDLDDDDDWWWLWDDLWWLLDWWWLWWWLWWDDWDWLWWLWLLWDDLDLWLWDDWDDDWWDWDLWLWDD
WWWDLLLLDDDDWWLLDWLDWWLDDWDLWDLWLDLDDWLDDLWLWWLWLWDDWLWWDDLDWWWLLDWLDWWWLDLDLLLLDDDDDD
DLDWWWDWLWWDWLWLLDLDWLWDLWDDDLDWDDLWDDWDWWWLDLLDLLLWLDDLWWWLDDLWLWDDLLWDLLWLDDWWLDLDWDD
WWDDLDDLWDWLLWLDWLWWDLDWDLDWWDDWDDWLWWLWWDWWDDLLDLLWWLDDLLLDWWDWLWWWLDLWDLWLWLWDWDDLLLLD
DLDLDDLWDLDLWDLWDDLLDDLDWWWWLLWWDDWWDLWLLWWDDDDLDDWWWDDLWWLLDLWDWWLDWLLLWWLLWWLDDDWWLDLWD
DWDWDWWDLLLDDWDWWDDWWDWWDDLLWWLDLWLDWWDWWLLWWDLDDLDLLLDDDDDWLLWDDWDLLLWDLWDLLLDLDLLWWWWLDD
DWDWWLLWDWWDDWLDWLDLWDDWDWWWWLLWLDWWLWWLDDWLWDWDLLDDDLLWDDWLDDDWWLLLWDDDLWLDDLLDDWWDDDLWLWD
LWDWDLLDLLWLWWWWLDLDLDDDLDLWLDDDWLWWDDWLDLWDWWWWDLDLWDWDLDWDLWLDLDDWLLLWWLDWWWDDDLWDWLDDWDWD
LWLDDDDDLLLDDWLLLWDWDWLLLDLDDWLDLLDDDLLDLDLLDLWLDLWDLDWWLDWDDWLDWDDDWDWLLWLWLDDLWDLLLLDLWDLWD
WDDDDWWLLLWDWWDWDLLWLLWDWWDWWLLWLDWLDLDLWDDDWDDLWWLDDWLWLDWWDWWWDDLWLLLWWDWDWDWDWWDLWDDWDWWLWD
DDLLDDWDLLWDLWLLLLWDLWDWWWDDDWWDWLDWDDWLWWLLLLLDLDWDDWLLWDDLLLDDWDDLDDDDLLWWLWWLWWLWWWLWLDDLLDD
WLDDWDWLLLWLDLLLLDLWDWWLWWLLDLWDWLWLWLWWLWLLWLDLLDDLWWDLLDDDDLWDLDDLDLLWDWDLDWLWWLDLLLLWWLWWLDDD
67 87
21 85
59 89
52 69
75 47
27 41
30 21
16 9
74 9
1 42
65 63
42 44
55 82
66 11
48 11
55 61
82 44
68 44
18 17
8 58
75 14
34 24
56 21
39 75
78 55
85 93
21 78
96 13
76 17
51 59
45 42
7 30
32 11
24 85
36 61
81 29
79 40
68 9
45 44
9 51
89 62
25 45
46 7
28 72
77 48
11 63
28 2
85 35
3 26
58 80
87 76
74 16
27 29
56 77
87 79
17 11
4 8
96 10
52 78
71 1
11 1
16 68
79 7
52 67
59 40
24 79
5 92
71 2
1 34
27 49
40 37
49 7
42 80
15 43
92 78
23 70
57 92
45 24
19 68
10 78
75 80
1 37
8 83
10 51
10 57
90 27
33 29
25 83
33 88
55 82
62 47
24 64
83 15
95 45
