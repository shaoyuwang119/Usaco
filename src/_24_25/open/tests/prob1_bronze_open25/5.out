1472
1472
1820
1472
1820
2160
2160
1295
1116
2160
1991
2327
567
1991
1820
1295
1295
1820
2160
2160
2327
1991
2160
2816
2160
1647
1472
2327
2816
1472
2160
2327
2492
1991
1295
2816
2816
1991
1116
1647
1116
1820
1991
1647
1991
1472
2492
1647
3287
1820
1991
2160
2492
1647
2655
1647
2816
1991
1820
1991
2160
1647
2327
1295
1647
3132
1295
2160
1472
1991
2160
2327
2655
1116
2160
1472
1647
2492
1991
2160
2160
2327
1820
1820
1820
1991
1991
935
2160
567
2160
1647
2160
2655
